PocketKit/Sources/PocketKit/Notifications/PocketBraze.swift

```diff
@@ -108,7 +107,22 @@ extension PocketBraze: BrazeInAppMessageUIDelegate {
         _ ui: BrazeInAppMessageUI,
         prepareWith context: inout BrazeInAppMessageUI.PresentationContext
     ) {
-        // Customize the in-app message presentation here using the context
+        // We need to set our scene for Braze because our UIKit + SwiftUI stuff does not seem to work with Braze's default
```

Comment:
```suggestion
        // We need to set our scene to show our local Braze in-app messaging because our UIKit + SwiftUI stuff does not seem to work with Braze's default
```

> @cyndichin that stuff is in the comment, can you help me make it more clear? That code creates a local in app message.
Ohh okay, added suggestion for the comment. I was more curious to understand why we need this local message when it worked previously without this code (we can also discuss live in Unleash meeting).

Reply:
@cyndichin that i can't answer, i dug through their sdk, and cant see why. But our codes been changing a lot. This is an area where im just accepting something changed, and this fixes it. 🤷‍♂️